### DISCUSSION TITLE
Shadow DOM

### DISCUSSION THREAD

**Discussion by pemrouz** [FortAwesome/Font-Awesome#20944] [Feature Requests]
Would be nice if you could prefix all rules with `body /deep/` by default so they cut across all Shadow DOM boundaries.

**Comment by chumtoadafuq** [comment]
yes, this is needed!

**Comment by chumtoadafuq** [comment]
For anyone else that ends up here... a workaround is to re-import the font-awesome CSS file within your shadow dom. You still need the main font-awesome stylesheet import in your root index.html.
@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css");

**Comment by stramel** [comment]
Shadow DOM piercing has been deprecated. `/deep/`in particular

**Comment by AStoker** [comment]
Hijacking this a bit, but Shadow DOM support is still suuuuper limited in Font Awesome (especially seeing this opened in 2014). It's kind of crazy that I end up having to duplicate all of font awesome for each shadow dom instance, resulting in rather large network transfers. 

Is there a better issue to track progress with Shadow DOM and Font Awesome?
